medusa/providers/torrent/html/bithdtv.py

```diff
@@ -147,8 +147,8 @@ def parse(self, data, mode):
                     if not all([title, download_url]):
                         continue
 
-                    seeders = try_int(cells[8].get_text(strip=True))
-                    leechers = try_int(cells[9].get_text(strip=True))
+                    seeders = try_int(cells[8].get_text(strip=True)) if len(cells) > 7 else 1
```

Comment:
It has nothing to do with Math. You have to think about what you are doing and what you want to do.